### PR TITLE
[NIGHTWACH E2E]: BAH - axeCheck must be updated in Applications / Edu Benefits / Tests / Edu Apply Wizard E2e Spec or test converted to Cypress

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe false positive violations were detected for color contrast. The best course of action now is to either update the axeCheck() call to use a custom config to ignore the color contrast rule (in progress) or a better approach will be to convert this test to Cypress.
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/21803## Testing done
Local testing `yarn cy:open`

## Acceptance criteria
- [x] All instances of the error are fixed
- [x] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations
- [x] Nightwatch e2e test Edu Apply Wizard E2e Spec passes axeCheck validation OR
- [x] Test is converted to Cypress and passes axeCheck validation

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
+// const E2eHelpers = require('../../../platform/testing/e2e/helpers');
+// const Timeouts = require('../../../platform/testing/e2e/timeouts.js');
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js:2]: Please remove commented code

File: src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+describe('Education Application Wizard', () => {
+  it('Goes through the education wizard', () => {
+    // Ensure education apply-wizard page renders.
+    // Open education apply wizard
+    cy.visit(`/education/how-to-apply/`)
+      .get('body', { timeout: 10000 })
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js:9]: Is ten seconds needed here and on the next few? Seems high. Do we want to define a constant for like a normal and/or long timeout? And use those throughout?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js (hunk 1 above)
Comment (ksantiagoBAH) [reply src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js]: removed

File: src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js (hunk 2 above)
Comment (ksantiagoBAH) [reply src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js:6]: updated to 1000 milliseconds

**Review by sonntag-adam — APPROVED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Pull request merged**